### DISCUSSION TITLE
too many open files

### DISCUSSION THREAD

**Discussion by jiajia-tao** [zouyaoji/vue-cesium#263] [Q&A]
大家好，我这边加载3dtile数据，使用了大量贴图图片，结果出现错误too many open files，请问是什么原因，又该如何解决？

**Comment by wewindy** [comment]
请上报错弹窗截图、控制台报错截图

**Reply by jiajia-tao** [reply]
![image](https://user-images.githubusercontent.com/70623063/186361710-ebc9c1e5-531e-45dc-8a69-2571ca2f2096.png)

**Reply by wewindy** [reply]
代码

**Reply by jiajia-tao** [reply]
![01d11900d5dc9d3a08d3af1c49290be](https://user-images.githubusercontent.com/70623063/186363611-6c99dd19-f1b3-47cb-a8fd-8c4320730784.jpg)

**Reply by jiajia-tao** [reply]
我引用的这个模型有3.02g

**Reply by wewindy** [reply]
似乎是数据集文件数量超过了 webpack 开发服务器伺服文件数量上限，建议把 3DTiles 转移至 web 服务器静态发布

**Reply by zouyaoji** [reply]
确实啊，这是工程都没运行起来，还以为啥呢，你把数据放nginx、node、tomcat、iis等任意的web服务的静态目录啊。
